github/ci/prow/files/docker-daemon-mirror.conf

```diff
@@ -0,0 +1,2 @@
+DOCKER_OPTS="${DOCKER_OPTS} --data-root=/docker-graph"
+DOCKER_OPTS="${DOCKER_OPTS} --registry-mirror=http://docker-mirror.kubevirt-prow.svc:5000 --insecure-registry=http://docker-mirror.kubevirt-prow.svc:5000"
```

Comment:
will the job use this registry instead of our internal `registry:5000`?

Reply:
That is just for docker itself. It pulls all kubevirtci images from the mirror in the datacenter instead of going to the internet. It should not interfere with bazel or our internal registry.